src/website/views/components/avatar.pug

```diff
@@ -158,30 +167,30 @@ p.-text
       .a-col.-w--2.-text--center
         div.a-avatar.-inverse AA
       .a-col.-w--2.-text--center
-        div.a-avatar.-mint AA
+        div.a-avatar.-secondary AA
```

Comment:
Let's remove the two white background examples. They don't meet accessibility requirements.

Reply:
done! I removed also Avatar Group secondary example